gcp_variant_transforms/libs/call_info_table_schema_generator.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates call_info table schema."""
+
+from apache_beam.io.gcp.internal.clients import bigquery
+
+from gcp_variant_transforms.libs import bigquery_util
+
+CALL_ID = 'call_id'
+CALL_NAME = 'call_name'
+FILE_PATH = 'file_path'
+TABLE_SUFFIX = '_call_info'
```

Comment:
Do you think we should include "_" in the suffix? I am asking this because for other suffixes (such as "chr1" or "variants" and "non_variants") we need to join it with the main name with a "_" added between. That's why I kind a feel we better remove it from here and instead user "_" to join them.